charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 4.4.0
-appVersion: 0.16.0
+version: 4.4.1
```

Comment:
Could you bump minor version instead of patch version? This is a new feature according to semver.

Reply:
Sure thing.